crates/ruff/src/checkers/ast.rs

```diff
@@ -3331,6 +3346,12 @@ where
                 if self.settings.rules.enabled(&Rule::RewriteUnicodeLiteral) {
                     pyupgrade::rules::rewrite_unicode_literal(self, expr, kind.as_deref());
                 }
+
+                if self.is_interface_definition {
+                    if self.settings.rules.enabled(&Rule::BanDocStringsInStubs) {
+                        flake8_pyi::rules::ban_doc_strings_in_stubs(self, expr);
+                    }
+                }
```

Comment:
Maybe this should instead be colocated with the other `let definition = docstrings::extraction::extract(...)` calls, on `FunctionDef` and `ClassDef`, to ensure they're "real" docstrings? (Does `flake8-pyi` flag this for _any_ standalone string? Or only actual docstrings?)

Reply:
oh good point

flake8-pyi flags any standalone string: https://github.com/PyCQA/flake8-pyi/blob/4212bec43dbc4020a59b90e2957c9488575e57ba/pyi.py#L1128-L1132

which probably isn't what we really want

Reply:
The other thing you could do, if you want, is put this logic in `check_definitions`, which iterates over all module, function, and class definitions, and you can check `definition.docstring.is_none()`. So you wouldn't need to put this in three places.

Reply:
even better!